### ISSUE TITLE
cntrl+click wont go at function

### ISSUE THREAD

**Issue by zaksnet** [godotengine/godot#31036]
**Godot version:**
3.2 Dev (also tested with 3.1)

**OS/device including version:**
Windows 10

**Issue description:**
Clicking on a function call when holding cntrl wont go at the function line as expected. It occassionaly works but i haven't figured a pattern to report so far. The functions you see in the Gif have been properly initiated in the script.

![cntrl+click](https://user-images.githubusercontent.com/1850856/62379517-c72a6900-b54f-11e9-99f8-d838b241a3c0.gif)

**Steps to reproduce:**
Hold cntrl and click on any function call.

**Comment by KoBeWi** [comment]
Related: #7080

Also there was an issue where Ctr+Click doesn't work when a method is defined after the call. I can't find it now (unless it wasn't reported?), but it could be that.

In short:
```
func _ready():
    meth()

func meth():
    pass
```
Ctrl+Click will fail here.

```
func meth():
    pass

func _ready():
    meth()
```
Here it should jump to the func above.

**Comment by 2shady4u** [comment]
Trying to solve this with PR #31112
All comments welcome!

**Comment by vini-guerrero** [comment]
Would it be possible to make this click event jump to a function that belongs to another script?

**Comment by NilsIrl** [comment]
This should be closed as it was fixed by #31843

**Comment by KoBeWi** [comment]
Yeah, solved by #31843
